src/ORM/Table.php

```diff
@@ -320,6 +329,7 @@ public function __construct(array $config = [])
         $this->_behaviors = $behaviors ?: new BehaviorRegistry();
         $this->_behaviors->setTable($this);
         $this->_associations = $associations ?: new AssociationCollection();
+        $this->queryFactory ??= new QueryFactory($this->getConnection(), $this);
```

Comment:
Coding style question: should we just do `$config['queryFactory'] ?? new QueryFactory()` here?

Reply:
We usually pass the same config array to `initialize()` as passed to the constructor without modification.